package.json

```diff
@@ -41,7 +41,7 @@
     "test:e2e": "node tests/e2e/testRunner.js --development"
   },
   "dependencies": {
-    "@expensify/react-native-web": "0.18.12",
+    "@expensify/react-native-web": "github:kidroca/react-native-web#8188a8b23d838dbe9fee9c856d3923bed6c0baf4",
```

Comment:
This is the complied code containing the flicker fix
After it's merged the package should be updated to point to the new version
- https://github.com/Expensify/react-native-web/pull/15

Reply:
should we throw a hold on this PR until https://github.com/Expensify/react-native-web/pull/15 is merged?